projects/worker-animal-pictures/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "example-worker-animal-pictures",
+    "author": "Adobe Inc.",
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "description": "Asset Compute example custom worker - Animal Pictures",
+    "repository": "adobe/example-worker-animal-pictures",
+    "dependencies": {
+        "@adobe/httptransfer": "^1.0.1",
+        "@adobe/asset-compute-sdk": "^1.0.2"
+    },
+    "devDependencies": {
+        "@adobe/aio-cli": "^3.4.1",
+        "@adobe/eslint-config-asset-compute": "^1.3.0",
+        "cross-os": "^1.3.0"
+    },
+    "scripts": {
+        "pretest": "cross-os pretest-cross-os",
+        "test": "aio asset-compute test-worker",
+        "posttest": "cross-os posttest-cross-os"
+    },
+    "cross-os": {
+        "pretest-cross-os": {
+            "darwin": "aio plugins:install @adobe/aio-cli-plugin-asset-compute",
+            "linux": "aio plugins:install @adobe/aio-cli-plugin-asset-compute",
+            "win32": "aio plugins:install @adobe/aio-cli-plugin-asset-compute"
+        },
+        "posttest-cross-os": {
+            "darwin": "eslint ./",
+            "linux": "eslint ./",
+            "win32": "eslint ..\\"
```

Comment:
Possibly remove - see .eslintrc.js comment.

Also this seems quite complex for an example. The examples should focus on how to write a Nui worker, and avoid any special npm complexities that might be confusing ("why does Nui need cross OS compilation" might be something that someone might ask from casually glancing over this package.json 😉 ).

Reply:
+1